bears/stylus/StylintBear.py

```diff
@@ -36,12 +38,182 @@ class StylintBear:
+            ``globalDupes: true`` is set, then it will also check for
+            duplicates globally (for root values only).
+        :param allow_efficient_properties:
+            Check for places where properties can be written more efficiently.
+        :param set_extend_preference:
+            Pass in either ``@extend`` or ``@extends`` and then enforce that.
+            Both are valid in Stylus. It doesn't really matter which one
+            you use.
+        :param indent_size:
+            This works in conjunction with ``max_selector_depth``. If you
+            indent with spaces this is the number of spaces you indent with.
+            If you use hard tabs, set this value to ``false``.
+        :param allow_leading_zero:
+            Enforce or disallow unnecessary leading zeroes on decimal points.
+        :param set_max_errors:
+            Set 'max' number of Errors.
+        :param set_max_warnings:
+            Set 'max' number of Warnings.
+        :param mixed_spaces_and_tabs:
+            Returns ``true`` if mixed spaces and tabs are found. If a number is
+            passed to indent_size, it assumes soft tabs (ie, spaces)
+            , and if ``false`` is passed to indentPref it assumes hard tabs.
+        :param set_naming_convention:
+            Enforce a particular naming convention when declaring classes, ids,
+            and variables. Throws a warning if you don't follow the convention.
+        :param set_strict_naming_convention:
+            By default, ``set_naming_convention`` only looks at variable names.
+            If ``set_strict_naming_convention`` is set to true,
+            ``set_naming_convention`` will  also look at class and id names.
+        :param allow_none_keyword:
+            If ``always`` check for places where ``none`` used instead of 0. If
+            ``never`` check for places where 0 could be used instead of none.
+        :param check_no_important_keyword:
+            If ``true``, show warning when ``!important`` is found.
+        :param allow_spaces_inside_parens:
+            Enforce or disallow use of extra spaces inside parens.
+        :param allow_placeholder:
+            Enforce extending placeholder vars when using ``@extend(s)``.
+        :param prefix_vars_with_dollar:
+            Enforce use of ``$`` when defining a variable. In Stylus using a
+            ``$`` when defining a variable is optional, but is a good idea
+            if you want to prevent ambiguity. Not including the ``$`` sets up
+            situations where you wonder: "Is this a variable or a value?"
+            For instance: padding ``$default`` is easier to understand than
+            padding ``default``.
+        :param allow_semicolons:
+            Enforce or disallow semicolons.
+        :param sort_order:
+            Enforce a particular sort order when declaring properties. Throws
+            a warning if you don't follow the order. If set to ``false``, allow
+            any order.
```

Comment:
and maybe instead of `false` let's take `none` or something like that. A "false sort order" is something different for me than "no false order"